Dockerfile

```diff
@@ -3,6 +3,6 @@ WORKDIR     /usr/src/zookeeper-exporter
 COPY        . /usr/src/zookeeper-exporter
 RUN         go build -v 
 
-FROM        alpine:3.6
+FROM        alpine:latest
```

Comment:
It's better to use explicit version instead of `latest`, so it's clear in any point of time which base image was used.
I've just checked and currently `latest` equals to `3.10`.